pkg/render/logstorage.go

```diff
@@ -403,13 +403,7 @@ func (es elasticsearchComponent) pvcTemplate() corev1.PersistentVolumeClaim {
 		Spec: corev1.PersistentVolumeClaimSpec{
 			AccessModes: []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
 			Resources: corev1.ResourceRequirements{
-				Limits: corev1.ResourceList{
-					"cpu":    resource.MustParse("2"),
-					"memory": resource.MustParse("3Gi"),
-				},
```

Comment:
Removing cpu and memory resources from PVC spec as they are not needed, only storage is valid for PVC.
https://kubernetes.io/docs/concepts/storage/persistent-volumes/#introduction

Reply:
Good to clean out the cruft.